### ISSUE TITLE
SW_PWR cannot detect button action

### ISSUE THREAD

**Issue by samchoy88** [PaxInstruments/t400-electronics#116]
![image](https://cloud.githubusercontent.com/assets/1327611/5580214/39c39330-9082-11e4-97af-fbff60f6a1a6.png)

I found the voltage at SW_PWR pin as below:
Button pressed: above 4V
Button released: above 1.2V

Arduino - digitalRead(SW_PWR) cannot detect the button released at 1.2V
I changed the R6 (10K) becomes 2K that can solve the issue.

**Comment by charlespax** [comment]
On Electronics version 0.9 hardware I changed R6 from whatever value it was to 2k. With a 2k resistor in place I measured the voltage between GND and the point indicated below. With the button unpressed I get 3.269 V. With the button pressed I get 3.287 V.
![screen shot 2015-01-06 at 10 49 15](https://cloud.githubusercontent.com/assets/281642/5623623/bb9b3cf0-9591-11e4-9fdf-ae9bdbe669ca.png)

Note that SW_PWR pin is Arduino digital 17, which is normally RXLED. I think our problem may be come from here. Perhaps the bootloader defines this as an output and sets it to HIGH. In the Blinkytape bootloader @cibomahto  disables RXLED and TXLED. Which bootloader did you flash onto these boards, the LilyPad Arduino USB in the Arduino IDE?
